### ISSUE TITLE
Shown Example Is Not Appropriate for arrays--callback-return

### ISSUE THREAD

**Issue by praveeno** [airbnb/javascript#1806]
[4.6 in Readme](https://github.com/airbnb/javascript#arrays--callback-return)
last example is shown as
   ```
    inbox.filter((msg) => {
      const { subject, author } = msg;
      if (subject === 'Mockingbird') {
        return author === 'Harper Lee';
      }
      return false;
    });
```

 but its not a example of `arrays--callback-return`, its illustrate `avoid of else`, when using return statement.

**Comment by ljharb** [comment]
I agree that the difference between the last two examples is about no-else-return, not just array-callback-return.

What would you suggest?

**Comment by abhinavsharma629** [comment]
Can you please assign me to this task sir?

**Comment by abhinavsharma629** [comment]
Just made a pull request for this .

**Comment by abhinavsharma629** [comment]
I just made a pull request regarding this issue sir.
Please have a look at it.
